collect_app/src/main/java/org/odk/collect/android/widgets/StringWidget.java

```diff
@@ -114,4 +124,66 @@ public void setDisplayValueFromModel() {
             Selection.setSelection(answerText.getText(), answerText.getText().toString().length());
         }
     }
+
+    private EditText getAnswerEditText(boolean readOnly, FormEntryPrompt prompt) {
```

Comment:
Huh. How was that over there?! Mysteries abound.

Reply:
Haha I was confused about this. Had to run all the tests just to convince myself moving it wasn't a bad idea.